proxylib/proxylib.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strconv"
 
 	"github.com/cilium/cilium/proxylib/accesslog"
+	_ "github.com/cilium/cilium/proxylib/binarymemcached"
```

Comment:
Can we move this package to `proxylib/memcached/binary`?

Reply:
done